src/site/content/en/blog/building-a-sidenav-component/index.md

```diff
@@ -0,0 +1,333 @@
+---
+layout: post
+title: Building a sidenav component
+subhead: A foundational overview of how to build a responsive slide out sidenav
+authors:
+  - adamargyle
+description: A foundational overview of how to build a responsive slide out sidenav
+date: 2021-01-16
+hero: hero.png
+thumbnail: thumb.png
+codelabs: codelab-building-a-sidenav-component
+tags:
+  - blog
+  - css
+  - dom
+  - javascript
+  - layout
+  - mobile
+  - ux
+---
+
+In this post I want to share with you how I prototyped a Sidenav component for the web that 
+is responsive, stateful, supports keyboard navigation, works with and without Javascript, 
+and works across browsers. Try the [demo](https://gui-challenges.web.app/sidenav/dist/).
+
+If you prefer video, here's a YouTube version of this post:
+
+{% YouTube 'ncYzTvEMCyE' %}
+
+## Overview
+
+It's tough building a responsive navigation system. Some users will be on a keyboard, 
+some will have powerful desktops, and some will visit from a small mobile device. 
+Everyone visiting should be able to open and close the menu. 
+
+<figure class="w-figure w-figure--fullbleed">
+  <video playsinline controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/atoms-sandbox.google.com.a.appspot.com/gui-sidenav/desktop-demo-1080p.mp4">
+  </video>
+  <figcaption class="w-figure">
+    Desktop to mobile responsive layout demo
+  </figcaption>
+</figure>
+
+<figure class="w-figure w-figure--fullbleed">
+  <video playsinline controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/atoms-sandbox.google.com.a.appspot.com/gui-sidenav/mobile-demo-1080p.mp4">
+  </video>
+  <figcaption class="w-figure">
+    Light and dark theme down on iOS and Android
+  </figcaption>
+</figure>
+
+## Picking tools {: #tools }
```

Comment:
Tools might not be the best word here. I think some readers might assume that you mean DevTools or build tools.

Reply:
nice, i agree. i've replaced usages of it with either tactics or web platform features. let me know what you think 👍🏻